### PR TITLE
Add missing option description for returning and silent

### PR DESCRIPTION
**Pull request by frogcjn**
Add missing option description for returning and silent

### CODE REVIEW

**Comment by mhegazy** [comment]
@frogcjn please address the CI failures.

**Comment by frogcjn** [comment]
Sorry. I've fixed this.

File: sequelize/sequelize.d.ts [hunk 1]
```diff
@@ -2907,6 +2883,46 @@ declare module "sequelize" {
 
         }
 
+        interface LoggingOptions {
+            logging?: boolean | Function;
```
Comment (mhegazy) [inline sequelize/sequelize.d.ts]: missing comments for `logging`. it used to be 
```ts
           /**
            * A function that gets executed while running the query to log the sql.
            */
```

Comment (frogcjn) [inline sequelize/sequelize.d.ts]: OK. Updated.

**Pull request merged**